app/app.ts

```diff
@@ -67,7 +67,7 @@ angular
   .component('logoutpage', logoutPage)
   .component('navigation', navigation)
   .constant('amTimeAgoConfig', {titleFormat: 'LLL'})
-  .run(function(IS_IOS) {
+  .run(function(IS_IOS: any) {
```

Comment:
Ignorance question, but is there a better annotation than `any` for things like this? It appears that `IS_IOS` is a `bool` so annotating with type `bool` would make sense.  I could be wrong, tho.

I've been wondering about similar things like `DataService: any`.  What is best practice for an annotation that should resolve to a specific service singleton?

Reply:
I could use boolean here.

Reply:
Largely we have ignored typing in this repo. I've added issue #593 to address this concern.

Reply:
Yeah, there is a lot of `any`.  I'd def like to see a file or two that somewhat follows best practices, so that if we wanted to fix it we could.